### PR TITLE
(#862) Add information on Group Details Screen

### PR DESCRIPTION
**Pull request by st3phhays**
## Description Of Changes
This adds information about the new Group Details screen that is added as part of Chocolatey Central Management 0.12.0.

## Motivation and Context
This is new information that needs documented as part of Chocolatey Central Management 0.12.0.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made
* [ ] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue
* #862

### CODE REVIEW

File: input/en-us/central-management/usage/website/groups.md [hunk 1]
```diff
@@ -69,6 +69,18 @@ You can enter a search term in the text field to filter results by typing in par
...
 
+## Viewing a Groups Details
+
+On the main Groups page, find the group you want to edit.
+You can enter a search term in the text field to filter results by typing in part of their Name or Description and clicking the :mag: button.
+Select the :gear: **Actions** button on the left-hand side of the group, and select **Details**.
+
+![Details menu entry in Group actions flyout menu](/assets/images/groups/ccm-groups-details.png)
+
+On the Group Details page, you'll find a searchable list of all Computers and Groups that currently in the Group, as well as a more verbose view of the Group information.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/groups.md:80]:
```suggestion
On the Group Details page, you'll find a searchable list of all Computers and Groups that are in the Group, as well as a more verbose view of the Group information.
```

File: input/en-us/central-management/usage/website/groups.md [hunk 2]
```diff
@@ -69,6 +69,18 @@ You can enter a search term in the text field to filter results by typing in par
 Similar to the [Edit Group](#editing-a-group) action, select the :gear: **Actions** button on the left-hand side of the group, and instead select **Delete**.
 You will be prompted to confirm the deletion.
 
+## Viewing a Groups Details
+
+On the main Groups page, find the group you want to edit.
```
Comment (AdmiringWorm) [inline input/en-us/central-management/usage/website/groups.md:74]: Wouldn't this be the group you want to `view`, not `edit`?

File: input/en-us/central-management/usage/website/groups.md [hunk 3]
```diff
@@ -69,6 +69,18 @@ You can enter a search term in the text field to filter results by typing in par
 Similar to the [Edit Group](#editing-a-group) action, select the :gear: **Actions** button on the left-hand side of the group, and instead select **Delete**.
 You will be prompted to confirm the deletion.
 
+## Viewing a Groups Details
```
Comment (vexx32) [inline input/en-us/central-management/usage/website/groups.md:72]:
```suggestion
## Viewing a Group's Details
```

**Review by vexx32 — COMMENTED** [review]
one minor nit, but the rest looks good I think :3

File: input/en-us/central-management/usage/website/groups.md (hunk 1 above)
Comment (corbob) [inline input/en-us/central-management/usage/website/groups.md:80]:
```suggestion
On the Group Details page, you'll find a searchable list of all Computers and Groups that currently in the Group, as well as a more detailed view of the Group information.
```

This feels like it'd be better to say Detailed as Verbose often has a negative connotation in this type of context.

**Pull request merged**